tf2_geometry_msgs/CMakeLists.txt

```
@@ -18,6 +18,16 @@ if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
 endif()
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
 
+# TODO(clalancette): This package doesn't directly depend on Eigen, but orocos-kdl
+# fails to properly export the dependency.  One possible solution would be to add
+# ${orocos_kdl_INCLUDE_DIRS} to the include directories, but that unfortunately causes
+# hard-coded paths to show up in Windows packaging builds.  Instead, depend on Eigen3
+# here to fix the missing export
+find_package(Eigen3 QUIET NO_MODULE)
+# Work around broken find module in AlmaLinux/RHEL eigen3-devel from PowerTools repo
+if(NOT Eigen3_FOUND)
+  find_package(Eigen3 REQUIRED)
+endif()
```

```suggestion
find_package(Eigen3 QUIET)
```

doesn't that just work?
Do we need to specify `NO_MODULE`?

---

> Do we need to specify NO_MODULE?

As I understand it, we need that whole above sequence for it to work on RHEL because of the bug in the Eigen packaging there.  But I didn't investigate this (I just copied it from elsewhere), maybe @sloretz can comment.